src/Symfony/Bridge/Twig/Tests/Command/LintCommandTest.php

```diff
@@ -74,11 +72,11 @@ public function testLintFileCompileTimeException()
     /**
      * @group legacy
      * @expectedDeprecation Passing a command name as the first argument of "Symfony\Bridge\Twig\Command\LintCommand::__construct()" is deprecated since Symfony 3.4 and support for it will be removed in 4.0. If the command was registered by convention, make it a service instead.
-     * @expectedException \RuntimeException
-     * @expectedExceptionMessage The Twig environment needs to be set.
      */
     public function testLegacyLintCommand()
     {
+        $this->expectException('RuntimeException');
```

Comment:
`::class` please. Also it seems you added the expectation always at the beginning of the function. this is mostly not correct like here. the exception is expected when calling a certain method like `execute`. This was the main reason why the annotation has been removed in phpunit.

Reply:
This PR is the raw result of an automate script that moves annotations to calls.

I'm fully agree with you on both points.
- I'll create a dedicated PR to add `::class` on all `expectException` (the ones introduce here + the rest of the code base)
- moving declaration of `expectException` to the right place cannot be automated and is too much effort (1295 occurences here)